src/gf/keldysh.jl

```diff
@@ -0,0 +1,92 @@
+struct KeldyshTimeGF{T, scalar} <: AbstractTimeGF{T, scalar}
+  grid::KeldyshTimeGrid
+  gtr::AntiHermitianStorage{T,scalar}
+  les::AntiHermitianStorage{T,scalar}
+  ξ::GFSignEnum
```

Comment:
Yeah this is not actually used in the `KeldyshTimeGF` code, but I left it since I thought you might want to keep track of if the GF is fermionic or bosonic.